### PR TITLE
lint-fixes

### PR DESCRIPTION
**Pull request by lionellbriones**

### CODE REVIEW

File: example/react-app/src/components/Console.tsx [hunk 1]
```diff
@@ -26,6 +26,7 @@ function Console() {
   }, [playgroundConsoleData]);
 
   return (
+    // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-noninteractive-element-interactions
```
Comment (chaitanyapotti) [inline example/react-app/src/components/Console.tsx:29]: don't disable stuff...

Comment (lionellbriones) [reply example/react-app/src/components/Console.tsx]: done.

File: example/react-app/src/components/Dialog.tsx [hunk 2]
```diff
@@ -8,22 +8,56 @@ type Props = {
...
+  }, [type]);
+
+  useEffect(() => {
+    const effectRef = dialogRef.current;
+
+    if (effectRef) {
+      effectRef.addEventListener("click", closeDialog);
+
+      return () => {
+        effectRef.removeEventListener("scroll", closeDialog);
```
Comment (chaitanyapotti) [inline example/react-app/src/components/Dialog.tsx:41]: removal should be of the same event name

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**